Apps/frontend/src/views/ControlPanelBottom.svelte

```diff
@@ -27,12 +26,11 @@
   } from "../labels";
   import { computeDisplayDeltaFromTimeSweep } from "../helper";
 
-
   export let controlPanelBottomHeight = 0;
 
   const computeDisplaySpeed = (value) => {
     let delta = computeDisplayDeltaFromTimeSweep(value);
-    return ((1000 * TIME_PER_DIV) / delta).toFixed(2) + " ms/div";
+    return (1 / delta).toFixed(2) + " s/div";
```

Comment:
Danke, dass du dir das mal angesehen hast. Ich bin mir jedoch nicht sicher, ob das richtig sein kann. Die Bestimmung des Display-Speed hängt jetzt nicht von der Update-Frequenz der Signale ab. Aber das müsste es ja. delta ist nur ein Faktor zwischen 0 und 2.

Beispiel:
Wenn jetzt unser Generator doppelt so schnell senden würde (z.B. EXPECTED_UPDATES_PER_SECOND =20_000), würde sich der Display-Speed nicht verändern. Aber nach meinem Verständnis sollte sich in dem Beispiel die Display-Speed dann halbieren.

Ich vermute, dass der Bug der noch drin ist, eher daher kommt dass in der Konstante CANVAS_WIDTH nicht die Anzahl der Pixel steht, sondern irgendeine andere Maßeinheit. Wir müssten also die Anzahl der Pixel/der Updates in der H

Mal sehen was die anderen sagen. Von mir aus kann man es wegen der wenigen verbleibenden Zeit auch so lassen, damit die Zeiteinheit am Demo-Day immerhin richtig aussieht, aber rein fachlich passt das so noch nicht (glaube ich).

Reply:
Ich hab es mal adressiert, und für die zukünftigen Bearbeiter einen Kommentar hinterlassen.